frontend/src/components/Common/Layout.tsx

```diff
@@ -1,7 +1,6 @@
 import { Suspense, useEffect } from 'react';
 import { Outlet, useLocation } from 'react-router-dom';
 import ReactGA from 'react-ga';
-import Banner from './Banner';
```

Comment:
Sorry there was some confusion, the intent was to prevent the banner from displaying in the DOM since we are no longer accepting applications, not removing the components entirely.

The way we've done it has just been to comment out, or remove the component from rendering.

Also I believe we still want the modal about the scheduler to appear still

Reply:
Got it!

About the scheduler modal, I checked with Matt and he said we should remove it. It's described in #610 as well :)

Reply:
Ah sounds good!

Reply:
Changes made :))